db/migrate/20160321161136_add_devise_to_users.rb

```diff
@@ -0,0 +1,49 @@
+class AddDeviseToUsers < ActiveRecord::Migration
+  def self.up
+    change_table(:users) do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
```

Comment:
To that end, it seems like this gem actually covers most of everything off of [WED-22](https://cs5500-jira.ccs.neu.edu/browse/WED-22) (at least the data model stuff) - unless I'm misreading the ticket or misinterpreting what this gem actually does?